source/index.md

```diff
@@ -539,9 +539,10 @@ payment method must be omitted.
   </dd>
 </dl>
 
-##### Method: `card`
+---
+##### **Method**: `card`
```

Comment:
I guess this was an attempt at a mini-fix, just to make it more usable.
My guess is I can find some better formatting, if need be.
Getting someone else to do it would be just fine by me :)